tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -259,6 +337,31 @@ def __validate_s2_records_are_related(self):
             t5s = reporting_year_cases.get(t5_model, [])
 
             if len(t4s) > 0:
+                if len(t4s) > 1:
+                    for record, schema in t4s[1:]:
+                        self.__generate_and_add_error(
+                            schema,
+                            record,
+                            field='RPT_MONTH_YEAR',
+                            msg=(
+                                f'There should only be one {t4_model_name} record  '
+                                f'for a RPT_MONTH_YEAR and CASE_NUMBER.'
+                            )
+                        )
+                        num_errors += 1
+                else:
+                    t4 = t4s[0]
+                    t4_record, t4_schema = t4
+                    closure_reason = getattr(t4_record, 'CLOSURE_REASON')
+
+                    if closure_reason == '01':
+                        num_errors += self.__validate_case_closure_employment(t4, t5s, (
+                            'At least one person on the case must have employment status = 1:Yes in the same month.'
```

Comment:
@jtimpe suggested language for this one from the DIGIT team:

`At least one person on the case must have employment status = 1:Yes in the same RPT_MONTH_YEAR since CLOSURE_REASON = 1:Employment/excess earnings`.